src/components/FileUpload/components/FilePreview.tsx

```diff
@@ -1,46 +1,36 @@
 import React from 'react'
-import { Document, Page } from 'react-pdf'
-import styled, { css } from 'styled-components'
+import { Document, Page } from 'react-pdf/dist/esm/entry.webpack'
```

Comment:
Are we really supposed to do this? `react-pdf/dist/esm/entry.webpack`?

Reply:
Yes, it actually looks like it's necessary - https://github.com/wojtekmaj/react-pdf#webpack

It has most probably changed in the latest version because I noticed that PDF previews stopped working since @konupek upgraded the version of this package. When using just `from 'react-pdf'`, on upload it fails to render the preview with error "Failed to load PDF file." and in the console it says `Setting up fake worker failed`.

The original import is broken both for Storybook "live" version and for bundled version, and this change fixes it.

Reply:
I see. 👍👍 interesting 

Could you please put there a comment with this link? 🙂